tests/pkg_semtech-loramac/main.c

```diff
@@ -20,19 +20,49 @@
 #include <string.h>
 #include <inttypes.h>
 
+#include "thread.h"
 #include "msg.h"
 #include "shell.h"
 #include "fmt.h"
 
 #include "net/loramac.h"
 #include "semtech_loramac.h"
 
+#define LORAMAC_RECV_MSG_QUEUE                   (4U)
```

Comment:
Why 4? Could it be less? Does it need to be more?

Reply:
This allows optimizing a bit the size in RAM while still allowing to have an asynchronous thread. Since this thread is not heavily loaded, I think it's fine to have a lower value. Previously it was 16 which is unnecessarily high (I have checked and most of RIOT examples uses 8 or 4).